src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionInfo.cs

```diff
@@ -52,6 +52,12 @@ public ConnectionInfo(ISqlConnectionFactory factory, string ownerUri, Connection
         public DbConnection SqlConnection { get; set; }
 
         /// <summary>
+        /// The connection to the SQL database that queries will be run against.
+        /// A single instance is created and re-used after the first query is run 
+        /// </summary>
+        public DbConnection QueryConnection { get; set; }
```

Comment:
>QueryConnection [](start = 28, length = 15)

don't we need one connection per SQL file?  The connectioninfo object is shared across multiple editor windows.